egs2/TEMPLATE/enh_asr1/scripts/utils/get_model_names.py

```diff
@@ -0,0 +1,7 @@
+from espnet_model_zoo.downloader import ModelDownloader
```

Comment:
Better to replace with a symbolic link to [egs2/TEMPLATE/asr1/scripts/utils/get_model_names.py](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/scripts/utils/get_model_names.py).